regen-network/regen-web
# feat: use credit class metadata in addition to sanity credit class data

## Description

Closes: regen-network/regen-registry#1428



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

C04 is not on Sanity staging but has some image stored in its metadata (as well as required name and description), see for instance:
https://deploy-preview-1935--regen-registry.netlify.app/credit-classes/C04
or https://deploy-preview-1935--regen-registry.netlify.app/project/C04-001
or any of the ecocredits / sell orders table.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
